src/olympia/addons/models.py

```diff
@@ -1268,6 +1268,18 @@ def get_featured_by_app(self):
             out[app].add(locale or None)
         return out
 
+    @cached_property
```

Comment:
Do we need it to be a `@cached_property`? Do we end up requesting that twice in a row somewhere for the same instance ? This can cause some headaches if it's cached early...

Reply:
it has to be a `@cached_property` so the ES serializer can set it's value based on what's in the index.

Reply:
Ah, I see.